### ISSUE TITLE
exchange one bp for another

### ISSUE THREAD

**Issue by shivadarwin** [DeepLabCut/DeepLabCut#1087]
Hey guys, first time here. Been using the software since 2.0 era and I am very grateful for all your hard work. Can I continue using a snapshot-xxx,xxx that is a well-trained and tracks great, use it on a different config file (new project) with one of the body parts replaced with another one, but in the same location? Will DLC2.2b8 account for the change in the bp name and re-train? If so, how many iterations? I just need to change the bp "implant" to "head", both are in the same location, and keep everything else the same.
Thanks

**Comment by jeylau** [comment]
Hey @shivadarwin, wouldn't it be simpler to keep using that great network you have and rename that bodypart afterwards in the data files?

**Comment by shivadarwin** [comment]
> Hey @shivadarwin, wouldn't it be simpler to keep using that great network you have and rename that bodypart afterwards in the data files?

Hi, thanks for the reply. Here are a couple of pics just for info purposes.
![no implant](https://user-images.githubusercontent.com/76235927/106315667-38c0a180-6231-11eb-868e-1be10d536a73.png)
![with implant](https://user-images.githubusercontent.com/76235927/106315674-3c542880-6231-11eb-82d7-853cacb73df3.png)
As you can see in the pics, the implant looks different and I was wondering if I can use the trained network (mouse with implant), run few more iterations on "no implant mouse" (just mark a spot between the ears as "head") with everything else the same. If the algorithm can re-orient itself to the new reference point, i.e. a pink head cap with flashing laser to a plain black spot between ears.  
Long story short, I went and re-trained a new network, just in case. Thanks for the support anyways, have a great day.

**Comment by shivadarwin** [comment]
And sorry for the late response. I thought github would send me a ping if my query was replied to, instead I got a message only after you closed the ticket.
